cmd/cl-adm/cmd/init/init.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 The ClusterLink Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package initcmd
```

Comment:
please have the directory path match the package name (ie. `cl-adm/cmd/initcmd`)

Reply:
nit: using init.go may suggest that this is the `init()` function for the package.

Reply:
@kfirtoledo perhaps just rename the command to some other name? e.g. `deploy`
I think that even if `init` was allowed, it it still not meaningful enough for the purpose of this command